### DISCUSSION TITLE
Rounding Corners on bpmn2.Activity

### DISCUSSION THREAD

**Discussion by gordon-brown** [clientIO/joint#2592] [Q&A]
### Introduction

Trying to make the corners on the activity shape to be a little less rounded because the shape looks more like an oblong then a rectangle in our stencil. I tried setting rx, ry properties in attrs/background but it look like its rendering the shape twice.

![Screenshot 2024-03-26 154846](https://github.com/clientIO/joint/assets/32515524/75b3d2ea-b6a8-4bd3-8c00-fd6bd3fec2b6)

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS+

**Comment by kumilingus** [comment]
Setting the `borderRadius` on `border` element should do the trick (currently only the same number is allowed for x and y).

```ts
element.attr('border/borderRadius', 5);
```
